includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -73,7 +73,7 @@ public function init( $sensei ) {
 		new \Sensei\Blocks\Course_Theme();
 
 		add_action( 'init', [ $this, 'register_post_meta' ] );
-		add_action( 'template_redirect', [ $this, 'maybe_redirect_to_course_theme' ] );
+		add_action( 'template_redirect', [ $this, 'maybe_redirect_to_correct_mode' ] );
```

Comment:
Just a naming thing, `correct` isn't very meaningful here. Not sure what the best would be, could even go along the lines of `ensure_learning_mode_url_prefix`?

Reply:
Yeah, wasn't really happy with that name either. Couldn't come up with anything better myself, but I think your suggestion sounds about right. Updated.